backend/src/v5/routes/teamspaces/projects/containers/containers.js

```diff
@@ -184,6 +203,94 @@ const establishRoutes = () => {
 	 */
 	router.get('/:container/stats', hasReadAccessToContainer, getContainerStats);
 
+	/**
+	 * @openapi
+	 * /teamspaces/{teamspace}/projects/{project}/containers/favourites:
+	 *   patch:
+	 *     description: Add containers to the user's favourites list
+	 *     tags: [Containers]
+	 *     operationId: appendContainers
+	 *     parameters:
+	 *       - teamspace:
+	 *         name: teamspace
+	 *         description: name of teamspace
+	 *         in: path
+	 *         required: true
+	 *         schema:
+	 *           type: string
+   	 *       - project:
+	 *         name: project
+	 *         description: ID of project
+	 *         in: path
+	 *         required: true
+	 *         schema:
+	 *           type: string
+	 *     requestBody:
+     *       content:
+     *         application/json:
+     *           schema:
+     *             type: object
+     *             properties:
+     *               containers:
+     *                 type: array
+	 *     responses:
+	 *       401:
+	 *         $ref: "#/components/responses/notLoggedIn"
+	 *       404:
+	 *         $ref: "#/components/responses/teamspaceNotFound"
+	 *       200:
+	 *         description: adds the containers found in the request body to the user's favourites list
+	 *         content:
```

Comment:
remove content, we don't need to send a response body if there's nothing to send.